src/lib/viewers/doc/_docBase.scss

```diff
@@ -49,6 +59,10 @@ $thumbnail-border-radius: 4px;
     .bp-thumbnail.bp-thumbnail-is-selected .bp-thumbnail-nav {
         border-color: $fours;
     }
+
+    &.bp-thumbnails-open .bp-content {
+        left: 201px;
```

Comment:
Would it be easier to manage if this was stored in a variable, just in case anything about the sidebar width changes?